### PR TITLE
Update info about TES3MP

### PR DESCRIPTION
**Pull request by testman42**
Changed back to active, added LAN as an option and defined content as "swappable" (as it can technically use OpenMW's Example Suite instead of Morrowind)

### CODE REVIEW

File: games/t.yaml [hunk 1]
```diff
@@ -145,26 +145,28 @@
...
 
-- name: TES3MP
-  images:
-  - https://i2.wp.com/openmw.org/wp-content/uploads/2017/07/hA3oZhX.png
-  lang:
-  - C++
-  license:
-  - GPL3
-  development: halted
+  name: TES3MP
```
Comment (cxong) [inline games/t.yaml:148]:
```suggestion
- name: TES3MP
```

File: games/t.yaml [hunk 2]
```diff
@@ -145,26 +145,28 @@
...
+  - LAN
+  content: swappable
+  lang:
+  - C++
+  license:
+  - GPL3
+  info: TES3MP is a project adding multiplayer functionality to OpenMW
+  updated: '2022-02-07'
+  images:
+  - 'https://i2.wp.com/openmw.org/wp-content/uploads/2017/07/hA3oZhX.png'
```
Comment (cxong) [inline games/t.yaml:167]: this image seems to be broken when I checked

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/t.yaml [hunk 3]
```diff
@@ -146,25 +146,27 @@
...
```
Comment (cxong) [inline games/t.yaml:167]:
```suggestion
  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/170828b.png
  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-2983-11-27-07h06m58s509.png
  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-4274-01-22-03h12m11s414.png
  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-2432-09-08-08h21m43s889.png
  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-7947-01-28-20h26m56s612.png
  video:
    moddb: 1907883
```

File: games/t.yaml [hunk 4]
```diff
@@ -146,25 +146,33 @@
...
+  info: TES3MP is a project adding multiplayer functionality to OpenMW
+  updated: '2022-02-07'
+  images:
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/170828b.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-2983-11-27-07h06m58s509.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-4274-01-22-03h12m11s414.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-2432-09-08-08h21m43s889.png
+  - https://media.moddb.com/cache/images/engines/1/1/929/thumb_620x2000/vlcsnap-7947-01-28-20h26m56s612.png
+  video:
+    moddb: 1907883
```
Comment (cxong) [inline games/t.yaml:173]: there should only be one video tag, with both youtube and moddb

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/t.yaml (hunk 4 above)
Comment (testman42) [reply games/t.yaml]: Done in [d97152b](https://github.com/opengaming/osgameclones/pull/1623/commits/d97152b75462157f6261f5135d0fbfb38dd1b5c3).

**Pull request merged**